src/module.ts

```diff
@@ -347,6 +400,43 @@ export default defineNuxtModule<ModuleOptions>({
         }
       )
 
+      if (options.search) {
+        if (options.search.indexedSearch) {
+          // Will return a string
+          nitroConfig.handlers.push({
+            method: 'get',
+            route: nuxt.options.dev
+              ? `${options.api.baseURL}/indexed-search.txt`
+              : `${options.api.baseURL}/indexed-search.${buildIntegrity}.txt`,
```

Comment:
Is there any specific reason for using `.txt` and not `.json`? 

Also, what do you think about merging `indexed-search.txt` and `search.json`. The result could vary based on `indexedSearch`

Reply:
Yes! But maybe I could use some headers to force Nitro to return a json without not in a `html`.

I do this to avoid that Nitro render the string in a `.html` file.

Reply:
Done. I remove txt and update `responseType` when fetching an indexed search.